test/correctness/mongo_model.py

```diff
@@ -495,7 +495,7 @@ def insert_one(self, dict):
     def insert_many(self, list):
         self.insert(list)
 
-    def find(self, query, fields=None):
+    def find(self, query, fields=None, batch_size=None):
```

Comment:
What's this `batch_size` used for?

Reply:
`batch_size` from driver converts to `numberToReturn` in `OP_QUERY`. This is kind of page size for large queries.